### ISSUE TITLE
Frequently read improvements

### ISSUE THREAD

**Issue by milen-prg** [sumatrapdfreader/sumatrapdf#4229]
**Is your feature request related to a problem? Please describe.**
The "Frequently read" list keeps history of book openings, but it would be better if it keep only one (the last), record for a book; when open again, the previous data record must be deleted. 

Additionally, the list is long, but only first 10 files can be seen and selected (2 rows by 5 columns). Make this scrollable with the whole history, please. The restriction of only 10 visible book titles is big problem with this otherwise excellent software!

**Describe the solution you'd like**
Make list "Frequently read" scrollable (to infinite or eventually with option for deleting records or all history) without repeating titles (most recent on the top).

**Comment by GitHubRulesOK** [comment]
The historic reason for 10 was there are only 10 keys  0 1 2 3 4 5 6 7 8 9  However I think the visual thumbs is now 20 !
![image](https://github.com/sumatrapdfreader/sumatrapdf/assets/12754913/dbeb52a5-bd6d-48bb-9956-aa04a5f5b02a)
and you can pick from the infinite list
![image](https://github.com/sumatrapdfreader/sumatrapdf/assets/12754913/0e04b3d6-9732-427c-83c0-2ae751ddfa7e)

**Comment by milen-prg** [comment]
@GitHubRulesOK , this is Awesome! But HOW to activate this. In my case is:

![image](https://github.com/sumatrapdfreader/sumatrapdf/assets/59639866/1062ec7c-d5a6-47f0-813d-95db543e0440)

Even if I install the pre-release. I don't see any options in settings (also advanced).

**Comment by GitHubRulesOK** [comment]
the number of rows is based on window height so I had to be almost full screen on 1440 high monitor to see all 4 rows the pulldown is the CTRL+K trigger that you can self assign to any key

**Comment by milen-prg** [comment]
I'm on laptop so it is with low height screen, this explains the only 10 items. But the Ctrl+K, showing all books list, is excellent! Thank, you! I never would find it as in the documentation is defined as opening Command palette, not the recent file list.
It will be more conventional if the Frequently Read Thumbnails on the Home Tab are scrollable to the infinite (as long is the history).

**Comment by alan-isaac** [comment]
> I'm on laptop so it is with low height screen, this explains the only 10 items. But the Ctrl+K, showing all books list, is excellent! Thank, you! I never would find it as in the documentation is defined as opening Command palette, not the recent file list. It will be more conventional if the Frequently Read Thumbnails on the Home Tab are scrollable to the infinite (as long is the history).

Agree: the name "Command palette" for Ctrl+K makes it hard to guess the intended functionality.
